src/components/Prompt/index.jsx

```diff
@@ -29,9 +29,9 @@ class Prompt extends React.Component {
     e.preventDefault();
     const { emptyInputAlertMessage, onClickCloseButton, onClickConfirmButton, dispatchShowToast } = this.props;
     const { promptInput: inputValue } = this.state;
-    if (inputValue.length === 0) {
+    if (inputValue.replace(/\s/gi, '').length === 0) {
```

Comment:
`trim`을 사용하는 방법도 있겠습니다.

```suggestion
    if (inputValue.trim().length === 0) {
```

Reply:
넵!